### DISCUSSION TITLE
Hide `List View` button / `Details` section

### DISCUSSION THREAD

**Discussion by ermix3** [awcodes/filament-curator#522] [Q&A]
I want to hide button `List View` ,
Anyone can help me please 🙏

I already make `php artisan vendor:publish ` to curator views. I can't find the button.

![image](https://github.com/user-attachments/assets/fcac92b9-e15e-441a-af01-755f56adabd9)

![image](https://github.com/user-attachments/assets/527dcf15-6aa2-4e2c-8721-b035997aaa5d)

**Comment by awcodes** [comment]
Those are in the resource and pages not the blade views. You can make your own resource and pages by extending the existing ones and changing the config file to use your versions.

**Comment by ermix3** [comment]
Ok. 👌
Thanks for time.
